modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/ProcessBuilderSourceReact.testcase

```diff
@@ -65,9 +65,29 @@ definition {
 	@ignore = "Test Stub"
 	@priority = 4
 	test CanNotPublishWithoutTitle {
+		task ("Given a Process Builder is created") {
+			property portal.acceptance = "true";
+			property test.name.skip.portal.instance = "Kaleodesigner#AddNewWorkflowDefinitionViaViewTab";
 
-		// TODO LRQA-74658 CanNotPublishWithoutTitle pending implementation
+			ApplicationsMenu.gotoPortlet(
+				category = "Workflow",
+				panel = "Applications",
+				portlet = "Process Builder");
 
+			Workflow.addNewDefinition(workflowDefinitionTitle = "");
+		}
+
+		task ("When published without a name") {
+			PortletEntry.publish();
+
+			Navigator.gotoBack();
+		}
+
+		task ("Then there should be an error message with the text 'Please name your workflow before publishing'") {
```

Comment:
For our tasks let's try something like 
Given a user navigates to Workflow Process Builder
When a user attempts to publish a definition with no name
Then an error to name workflow before publishing will prevent this action. 

In the "Then" task we can assert the error message and assert the new workflow is not created.